website/content/docs/connect/config-entries/service-router.mdx

```diff
@@ -1372,6 +1382,75 @@ spec:
 </Tab>
 </Tabs>
 
+### Path prefix matching with case insensitive
+
+The following example routes HTTP requests for the `web` service to a service named `admin` when they have `/admin` or `/Admin` at the start of their path.
+
+<Tabs>
+<Tab heading="HCL" group="hcl">
+
+```hcl
+Kind = "service-router"
+Name = "web"
+Routes = [
+  {
+    Match {
+      HTTP {
+        PathPrefix = "/Admin"
```

Comment:
Is the `CaseInsensitive = true` field supposed to be set in these examples?

Reply:
Here it is https://github.com/hashicorp/consul/pull/19647/commits/0b15bec93ed8ff1e6dbfb3d7f6c16ebef0e06bbd